modules/nested-accordion/widgets/nested-accordion.php

```diff
@@ -44,6 +44,10 @@ public function get_keywords() {
 		return [ 'nested', 'tabs', 'accordion', 'toggle' ];
 	}
 
+	public function get_style_depends(): array {
+		return [ 'nested-accordion' ];
```

Comment:
@rami-elementor - is it ok that it's going to be loaded in the footer? (layout shift etc..)

Reply:
```suggestion
		return [ $this->get_name() ];
```